src_common/hash_bytes.c

```diff
@@ -19,8 +17,6 @@ void hash_nbytes(const uint8_t *const bytes_ptr, uint8_t n, cx_hash_t *const has
  * @param[in] byte byte to hash
  * @param[in] hash_ctx pointer to the hashing context
  */
-void hash_byte(uint8_t byte, cx_hash_t *const hash_ctx) {
+void hash_byte(uint8_t byte, cx_hash_t *hash_ctx) {
```

Comment:
`feed_byte_to_hash()`

Reply:
Can be looked at in another PR, since these functions predate this PR and are used elsewhere in the app.